tests/crypto/pbkdf2_vectors.py

```diff
@@ -14,7 +14,6 @@
 
 
 def print_vector(pwd, salt, iteration, dkLen):
-
```

Comment:
does it about code format?

Reply:
Yes. Code format tool removed a blank line.